io.unobserved.espansoGUI.json

```diff
@@ -28,7 +28,7 @@
         {
           "type": "archive",
           "url": "https://github.com/unobserved-io/espanso-gui/releases/download/2023.09.15/espanso-gui-2023.09.15.tar.xz",
-          "sha256": "82376cddfd58d02fa3fbe5321e95ab0c7d5d8829c7c6ba09ca9d27926e4e34b6"
+          "sha256": "f34a773d6ee604a13915c7c2bec22994cdbbbe1c40dfa21c850080923479a236"
```

Comment:
"retagging" is a bad practice.

Reply:
Sorry, I'm not sure what you mean, could you explain?

Reply:
you regenerated a tarball with the same name. That's retagging. It's a bad practice (nothing to do with Flathub). It should be a new release.

Reply:
Oh, okay, thank you.